src/transforms/filter.jl

```diff
@@ -69,22 +65,45 @@ DropMissing(cols::T...) where {T<:ColSelector} =
 
 isrevertible(::Type{<:DropMissing}) = true
 
-_ftrans(::DropMissing{Colon}, table) =
+# ftrans
+_ftrans(::DropMissing{Colon}, cols) =
   Filter(row -> all(!ismissing, row))
 
-function _ftrans(transform::DropMissing, table)
-  allcols = Tables.columnnames(table)
-  cols = _filter(transform.colspec, allcols)
+_ftrans(::DropMissing, cols) =
   Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
-end
+
+# nonmissing 
+_nonmissing(::Type{T}, c) where {T} = c
+_nonmissing(::Type{Union{Missing,T}}, c) where {T} =
+  collect(T, c)
```

Comment:
```suggestion
_nonmissing(::Type{Union{Missing,T}}, c) where {T} = collect(T, c)
```

Reply:
OK

Reply:
Please use the like button if you want to express agreement. Don`t need to reply OK in every suggestion because these are sent to everyone's mail box.

Reply:
Donde with [7db5e28](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/7db5e28ea9f808a8a4a1f1426f7ccf7b866f034c)